aces/Loris
# [login] Add code sanitizer before dangerouslySetInnerHTML

## Brief summary of changes
This PR adds DOMPurify to sanitize the code that is passed through dangerouslySetInnerHTML to avoid XSS vulnerabilities. 

#### Testing instructions (if applicable)

1. Run `npm install dompurify`
2. Before checking out this PR, edit and save the description field of GUI tab in the configuration module:
`<h3 onclick="alert('Attack message test')">Example Study Description</h3>` (or something similar)
3. Logout and try clicking on "Example Study Description" from the login page. You should see a message pop-up
4. Checkout this PR and run make dev
5. Try again to click on  "Example Study Description" from the login page. You should now see no pop-up. 
6. Make sure that the intended html is still rendering (bold, titles, etc.)
